### DISCUSSION TITLE
Сайт для поиска книг

### DISCUSSION THREAD

**Discussion by audetv** [audetv/cohesion#7] [General]
Куда разместим фронтенд для поиска книг? Пока не важно?
На подlомен svodd.ru?

например, lib.svodd.ru?

Какие будут предложения?

Я, конечно, завтра его ещё не запущу, но уже планирую заняться.

**Comment by iprst** [comment]
> lib.svodd.ru

Да, я думал именно в таком ключе, во всяком случае сейчас. Просто поддомен для обкатки модели.

**Reply by iprst** [reply]
Кстати, поскольку у нас сейчас две «библиотеки» — военно-историческая 8480 и географическая 200, можно сразу два поддомена сделать — lib.svodd.ru и geo.svodd.ru для тестирования разных идей: в первой мы будем обкатывать «большую» библиотеку и узнавать соответствующие проблемы, а во второй мы будем изучать «узкоспециальную» библиотеку, структура которой пока нам неизвестна в силу сложности размещения в голове всех представлений одновременно. Так движение в сторону различения будет более структурированным.

**Reply by audetv** [reply]
Да, хорошо, логично, принято.
